### PR TITLE
Fyndor#3 working out the data models

### PR DESCRIPTION
**Pull request by fyndor**
It looks like this will be the starting point for the data models

### CODE REVIEW

File: BernieApp.Common/Models/ArticleData.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        [JsonProperty(PropertyName = "@timestamp")]
+        public DateTime TimeStamp { get; set; }
+
+        [JsonProperty(PropertyName = "article_type")]
+        public string ArticleType { get; set; }
+
+        public int ExecutionTime { get; set; }
+
+        public int HttpStatusCode { get; set; }
```
Comment (atticoos) [inline BernieApp.Common/Models/ArticleData.cs:16]: Could probably have a class even higher up for any `RemoteData` entity, yea?

**Comment by fyndor** [comment]
@ajwhite Could you rephrase what you are asking?  What you are seeing is part of the data under the _source property in the issues / news responses.  for the full model of the _source property, i decided to inherit from this ArticleData and tack on the specific data for the requested datatype (only requesting one type per request).  Was trying to be flexible enough to deal with other versions of the _source property.  

Since I am trying to see if I can come up a model structure that is flexible enough to work will all the different es4bs endpoints, it appears I oversimplified the model a bit now that I look at the response from http://search.berniesanders.tech/geolocation/postal/_search?q=78218 

I will be updating it to try and reflect the endpoints pinned in the channel info for #elastic-search-es4bs  In the end they may end up adding endpoints that force me not use one response model structure for all the endpoints, but at first glance it appears right now I should be able to come up with a unified model for all responses

**Comment by atticoos** [comment]
I figured this was the status code in the HTTP response and assumed that any data model from a remote source might share this property and could see it being higher up the inheritance tree.

Not sure how things are structured though, so don't mind me!

**Comment by fyndor** [comment]
Ah.  Yea it is the kind of field you think it is, but it is not for THIS request.  I am not 100% sure where it comes from, my guess is as good as yours.  I'm thinking it is the status the server got (that I am calling to) when it tried to get data from it's own source.  Since this data is scraped and second hand I am guess it is from the originating request.

**Comment by fyndor** [comment]
One of the questions I was eventually going to ask someone was how I should deal with it if that code was 404 but my own request suceeeded.  To the end user it should basically be the same I guess because you don't have data for them.  But I didn't know if I should respond in some certain way.

**Comment by atticoos** [comment]
We'll want something in place. We can smooth out those edges in good time :+1:
